server/service/transport.go

```diff
@@ -410,19 +471,24 @@ func carveListOptionsFromRequest(r *http.Request) (fleet.CarveListOptions, error
 		return fleet.CarveListOptions{}, err
 	}
 
-	copt := fleet.CarveListOptions{ListOptions: opt}
+	carveOpts := fleet.CarveListOptions{ListOptions: opt}
 
+	// NOTE: This 'expired' parameter is NOT documented in rest-api.md
```

Comment:
@rachaelshaw Why are carve endpoint parameters (page, per_page, etc.) not documented in rest-api.md?

Reply:
Before my time so I can't say for sure, but if the carve endpoint supports pagination, my guess is we forgot to document that feature when it was shipped (because the example response doesn't show any pagination info either, and normally it'd have the number of pages and all that.)